.travis.yml

```diff
@@ -12,7 +12,7 @@ matrix:
       language: generic
       services:
         - docker
-      env: PYTHON=2.7 DOCKER_IMAGE=quay.io/pypa/manylinux1_x86_64 PYVER=cp27-cp27mu
+      env: PYTHON=2.7 DOCKER_IMAGE=quay.io/pypa/manylinux1_x86_64 PYVER=cp27-cp27mu PYVER2=cp27-cp27m
```

Comment:
May look at https://stackoverflow.com/questions/918886/how-do-i-split-a-string-on-a-delimiter-in-bash . Delimiter is recommended to be ``:``.